.github/workflows/KeyExists_build_and_test_on_main.yml

```
@@ -0,0 +1,18 @@
+name: KeyExists_build_main
+
+on:
+  push:
+    branches:    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)